roles/common/tasks/kerberos.yml

```diff
@@ -7,6 +7,17 @@
     state: present
   when: ansible_distribution == 'RedHat'
 
+- name: Update apt cache.
```

Comment:
Maybe?  This was the only file that cared about apt at all in the common role, which is why I put it here.

I don't mind moving it if you think it should be though.